draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -93,11 +93,11 @@ mechanisms defined in this document can be used for other use cases as well.
 In an Issuer-signed SD-JWT, claims can be hidden, but cryptographically
```

Comment:
```suggestion
In an SD-JWT, claims can be hidden, but cryptographically
```
In preparation for potentially calling the first part Issuer-Signed JWT or something similar.